static/src/App.vue

```diff
@@ -55,9 +55,7 @@ export default defineComponent({
         v-model:isAttentive="isAttentive"
         v-model:isTrained="isTrained"
       />
-      <i-label-group block>
-        <UserInformationLabel :username="user" />
-      </i-label-group>
+      <UserInformationLabel :username="user" />
```

Comment:
Is there any particular reason for removing it? I think Jasper mentioned it being somewhat important in another PR🤔